app/src/main/java/com/mashup/lastgarden/ui/scent/ScentPagingAdapter.kt

```diff
@@ -0,0 +1,76 @@
+package com.mashup.lastgarden.ui.scent
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.paging.PagingDataAdapter
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.mashup.base.extensions.loadImage
+import com.mashup.base.image.GlideRequests
+import com.mashup.lastgarden.R
+import com.mashup.lastgarden.data.vo.Story
+import com.mashup.lastgarden.databinding.ItemScentBinding
+import com.mashup.lastgarden.utils.convertDate
+import com.mashup.lastgarden.utils.formatNumber
+import com.mashup.lastgarden.utils.formatPageCount
+
+class ScentPagingAdapter(
+    private val glideRequests: GlideRequests,
+    private val listener: OnClickListener? = null
+) : PagingDataAdapter<Story, ScentPagingAdapter.ScentViewHolder>(DIFF_CALLBACK) {
+
+    companion object {
+        private val DIFF_CALLBACK = object : DiffUtil.ItemCallback<Story>() {
+            override fun areItemsTheSame(
+                oldItem: Story,
+                newItem: Story
+            ): Boolean = oldItem.storyId == newItem.storyId
+
+            override fun areContentsTheSame(
+                oldItem: Story,
+                newItem: Story
+            ): Boolean = oldItem == newItem
+        }
+    }
+
+    override fun onCreateViewHolder(viewGroup: ViewGroup, viewType: Int): ScentViewHolder {
+        return ScentViewHolder(
+            ItemScentBinding.inflate(
+                LayoutInflater.from(viewGroup.context),
+                viewGroup,
+                false
+            )
+        )
+    }
+
+    class ScentViewHolder(val binding: ItemScentBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+    }
```

Comment:
body 지워주시고 포매팅을
```suggestion
    class ScentViewHolder(
    val binding: ItemScentBinding
    ) :RecyclerView.ViewHolder(binding.root) 
```
이렇게 해주세요